src/browser/main.coffee

```diff
@@ -77,14 +77,9 @@ setupAtomHome = ->
     atomHome = fs.realpathSync(atomHome)
   process.env.ATOM_HOME = atomHome
 
-setupCoffeeCache = ->
-  CoffeeCache = require 'coffee-cash'
-  cacheDir = path.join(process.env.ATOM_HOME, 'compile-cache')
-  # Use separate compile cache when sudo'ing as root to avoid permission issues
-  if process.env.USER is 'root' and process.env.SUDO_USER and process.env.SUDO_USER isnt process.env.USER
-    cacheDir = path.join(cacheDir, 'root')
-  CoffeeCache.setCacheDirectory(path.join(cacheDir, 'coffee'))
-  CoffeeCache.register()
+setupCompileCache = ->
+  compileCache = require('../compile-cache')
```

Comment:
On my MBP:

```sh
node -p 't1 = new Date().getTime(); require("coffee-cash"); new Date().getTime() - t1'
24
node -p 't1 = new Date().getTime(); require("./src/compile-cache"); new Date().getTime() - t1'
38
```

So there's a 16ms increase without asar. I would expect a smaller number when using the asar bundle. In my opinion, we should try to get that down, but not by maintaining what is now a completely duplicated code-path. We should just convert the browser-side code to JS at some point soon and eliminate that whole chunk of time.